modules/storage-ephemeral-storage-types.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// storage/understanding-persistent-storage.adoc[leveloffset=+1]
+
+[id=storage-ephemeral-storage-types{context}]
+= Types of ephemeral storage
+
+Ephemeral local storage is always made available in the primary
+partition. There are two basic ways of creating the primary
+partition: root and runtime.
+
+== Root
+
+This partition holds the kubelet’s root directory, `/var/lib/origin/` by
+default, and `/var/log/` directory. This partition may be shared between user
+Pods, OS, and Kubernetes system daemons. This partition can be consumed by Pods
+via EmptyDir volumes, container logs, image layers, and container writable
+layers. Kubelet manages shared access and isolation of this partition. This
+partition is ephemeral, and applications cannot expect any performance SLAs,
+such as disk IOPS, from this partition.
+
+== Runtime
+
+This is an optional partition that runtimes can use for overlay
+file systems. {product-title} attempts to identify and provide
+shared access along with isolation to this partition. Container image
+layers and writable layers are stored here. If the runtime partition
+exists, the `root` partition does not hold any image layer or other writable storage.
+
+[NOTE]
+====
+When you use DeviceMapper to provide runtime storage, a container's copy-on-write layer is not accounted for in ephemeral storage management. Use overlay storage to monitor this ephemeral storage.
+====
```

Comment:
This no longer applies to 4.x - everything is `overlay`

Reply:
OK, removed the note altogether.